pyecharts/chart.py

```diff
@@ -373,6 +379,10 @@ def _config_components(
         # dataZoom，勿改动
         if kwargs.get("is_datazoom_show", None) is True:
             self._option.update(dataZoom=chart["datazoom"])
+        if kwargs.get("is_datazoom_extra_show", None) is True:
```

Comment:
Use `if kwargs.get("is_datazoom_extra_show", False) is True:`  to explicitly specify its type as `bool`

Reply:
DONE